src/repository/workflow_repository.go

```diff
@@ -0,0 +1,66 @@
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"github.com/input-output-hk/cicero/src/model"
+	"github.com/uptrace/bun"
+)
+
+type WorkflowRepository interface {
+	GetAllByName(string)([]model.WorkflowInstance, error)
+	GetByNameAndId(string, uint64)(model.WorkflowInstance, error)
+	GetById(uint64)(model.WorkflowInstance, error)
+	Save(*bun.Tx, *model.WorkflowInstance)(sql.Result, error)
+	Update(*bun.Tx, uint64, *model.WorkflowInstance)(sql.Result, error)
+}
+
+type workflowRepository struct {
+	DB *bun.DB
+}
+
+func NewWorkflowRepository(db *bun.DB) WorkflowRepository {
+	return workflowRepository{DB: db}
+}
+
+//TODO: why by name? Id is primary key
```

Comment:
Yes, the name parameter has been removed upstream in `WorkflowForInstance()` which was the sole usage of this function